README.md

```diff
@@ -29,3 +29,34 @@ mailing list as well.
 
 ### Other useful links
 * [Public group email archive](https://lists.w3.org/Archives/Public/public-vc-wg/)
+
+## Process Overview for VC Data Model Pull Requests
+1. For now, we will focus only on merging new errata PRs into this repository,
+but encourage activity related to new features.
+2. Once a PR is opened, chairs and editors make judgement call on whether
+changes are substantive or editorial.
+```
+Editorial   => mark with "editorial" tag
+               would be merged into branch "v1.1"
+Substantive => mark with "substantial" tag
+    Bugfix       |=> goes into separate branch "v1.2"
+    New Features |=> stays as an open PR?
+```
```

Comment:
```suggestion
<dl>
  <dt>Editorial</dt>
  <dd>Mark with "editorial" tag, merge into branch "v1.1"</dd>
  <dt>Substantive</dt>
  <dd>Mark with "substantial" tag</dd>
  <dt>Bugfixes - merge into separate branch "v1.2". New Features stay around as an open PR.</dd>
</dl>
```

Per @gkellogg's comments, here's a `dl` suggestion.

Reply:
Opinions about mixing HTML with Markdown vary. I don't have strong feelings about it, and @msporny's suggestion seems fine, as far as it goes. My concern here is only that the definition list (or whatever) nest properly within/beneath step 2, rather than confusingly being on the same visual level as steps 2 and 3.

Reply:
I agree on the formatting concerns, @TallTed -- we can rewrite the entire list in HTML if Markdown doesn't do the right thing. I hesitate to experiment w/ this in a PR. Once this is merged, I'll ensure it's properly formatted.